targets/mbed/source/jerry_extapi.cpp

```diff
@@ -86,34 +86,32 @@ static bool
 register_native_function (const char* name,
                           jerry_external_handler_t handler)
 {
-  jerry_api_object_t *global_obj_p;
-  jerry_api_object_t *reg_func_p;
-  jerry_api_value_t reg_value;
-  bool bok;
+  jerry_value_t global_object_val;
+  jerry_value_t reg_function;
+  jerry_value_t reg_value;
+  jerry_value_t bok;
 
-  global_obj_p = jerry_api_get_global ();
-  reg_func_p = jerry_api_create_external_function (handler);
+  global_object_val = jerry_get_global_object ();
+  reg_function = jerry_create_external_function (handler);
 
-  if (!(reg_func_p != NULL
-      && jerry_api_is_function (reg_func_p)
-      && jerry_api_is_constructor (reg_func_p)))
+  if (!(reg_function != 0
+      && jerry_value_is_function (reg_function)
+      && jerry_value_is_constructor (reg_function)))
   {
     printf ("Error: create_external_function failed !!!\r\n");
-    jerry_api_release_object (global_obj_p);
+    jerry_release_value (global_object_val);
     return false;
   }
 
-  jerry_api_acquire_object (reg_func_p);
-  reg_value.type = JERRY_API_DATA_TYPE_OBJECT;
-  reg_value.u.v_object = reg_func_p;
+  reg_value = jerry_get_global_object ();
+  jerry_value_t jerry_name = jerry_create_string((jerry_char_t*) name);
 
-  bok = jerry_api_set_object_field_value (global_obj_p,
-                                          (jerry_api_char_t *) name,
-                                          &reg_value);
+  bok = jerry_set_property (global_object_val,
+                            jerry_name,
+                            reg_value);
 
-  jerry_api_release_value (&reg_value);
-  jerry_api_release_object (reg_func_p);
-  jerry_api_release_object (global_obj_p);
+  jerry_release_value (reg_value);
+  jerry_release_value (global_object_val);
 
   if (!bok)
```

Comment:
@polaroi8d, I think 'b' is for boolean. It is similar to 'is_ok' variable, but the type of 'bok' is changed. Use something else, like `result`, `result_value`, etc.